react/src/pages/AccountsComponent.js

```diff
@@ -78,69 +160,162 @@ const AccountInfoLeft = ({ header, info, foursightUrl }) => {
+                            style={{marginLeft:"4pt"}} />
+                    </>:<>{Char.EmptySet}</>}
+                </td>
+            </tr>
+        }
+        { info.get("foursight.redis_url") &&
+            <tr style={{fontSize:"small"}}>
+                <td style={{paddingRight:"10pt"}}>
+                    Redis:
+                </td>
+                <td>
+                    {info.get("foursight.redis_url")}
+                    <ExternalLink
+                        href={`https://us-east-1.console.aws.amazon.com/memorydb/home?region=us-east-1#/clusters`}
+                        style={{marginLeft:"4pt"}} />
+                </td>
+            </tr>
+        }
+        { info.get("foursight.sqs_url") &&
+            <tr style={{fontSize:"small"}}>
+                <td style={{paddingRight:"10pt"}}>
+                    SQS:
+                </td>
+                <td>
+                    {info.get("foursight.sqs_url")}
+                    <ExternalLink
+                        href={`https://us-east-1.console.aws.amazon.com/sqs/v2/home?region=us-east-1#/queues/${encodeURIComponent(info.get('foursight.sqs_url'))}`}
+                        style={{marginLeft:"4pt"}} />
+                </td>
+            </tr>
+        }
         <tr><td style={{paddingTop:"4pt"}} /></tr>
         <tr><td colSpan="2" style={{borderTop:"1px dotted"}} /></tr>
         <tr><td style={{paddingTop:"4pt"}} /></tr>
+
+        { info.get("foursight.identity") == info.get("portal.identity") ? <>
         <tr style={{fontSize:"small"}}>
             <td style={{paddingRight:"10pt"}}>
                 Identity:
             </td>
             <td style={{whiteSpace:"break-spaces",wordBreak:"break-all"}}>
                 { info.get("foursight.identity") ? <>
                     {info.get("foursight.identity")}
+                    <ExternalLink
+                        href={`${info.get("foursight.url")}/react/${info.get("foursight.default_env.name")}/aws/infrastructure?secrets=${info.get("foursight.identity")}`}
+                        style={{marginLeft:"6pt"}} />
                 </>:<>{Char.EmptySet}</>}
             </td>
         </tr>
+        </>:<>
+        <tr style={{fontSize:"small"}}>
+            <td style={{paddingRight:"10pt"}}>
+                Foursight Identity:
+            </td>
+            <td style={{whiteSpace:"break-spaces",wordBreak:"break-all"}}>
+                { info.get("foursight.identity") ? <>
+                    {info.get("foursight.identity")}
+                    <ExternalLink
+                        href={`${info.get("foursight.url")}/react/${info.get("foursight.default_env.name")}/aws/infrastructure?secrets=${info.get("foursight.identity")}`}
+                        style={{marginLeft:"6pt"}} />
+                </>:<>{Char.EmptySet}</>}
+            </td>
+        </tr>
+        <tr style={{fontSize:"small"}}>
```

Comment:
Same deal here should be able to refactor into helper

Reply:
Yes, refactoring some of this out, thanks.